ethereum/truffle-config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       provider: () => {
         return new HDWalletProvider(
           process.env.MNEMONIC,
-          "https://goerli.infura.io/v3/" + process.env.INFURA_KEY
+          "wss://goerli.infura.io/ws/v3/" + process.env.INFURA_KEY
```

Comment:
Interesting change. `http` does not work well now?

Reply:
Yeah, it times out very often with `https`, changing it to `wss` makes it very stable.

Reply:
Probably due to rate limit of infura